lib/pure/asyncfile.nim

```diff
@@ -466,6 +466,38 @@ proc write*(f: AsyncFile, data: string): Future[void] =
       addWrite(f.fd, cb)
   return retFuture
 
+proc len*(f: AsyncFile): int64 =
```

Comment:
There is ``proc getFileSize*(f: File): int64 {.tags: [ReadIOEffect], benign.}`` in system and sysio, I could mark ``getFileSize`` as deprecated and define ``len`` for ``File``, or add ``getFileSize`` to ``AsyncFile``.